src/client/containers/fetchMethod/fetch.js

```diff
@@ -0,0 +1,23 @@
+import { getUserToken } from '../../firebase/googleAuth';
+import { ApiError } from '../../ErrorBoundary';
+
+export async function fetchFromDb(endpoint, fetchMethod, postBody = {}) {
+  const token = await getUserToken();
```

Comment:
In case the token needs to be refreshed and the request fails... does this need a `try` & `catch`?